build.xml

```diff
@@ -184,7 +198,6 @@
         </if>
     </target>
 
-
```

Comment:
I generally keep two blank lines between `<target>` tags, because XML can be hard to scan and the extra whitespace helps.